telegram/_chatmember.py

```diff
@@ -159,6 +159,10 @@ class ChatMemberAdministrator(ChatMember):
     Represents a chat member that has some additional privileges.
 
     .. versionadded:: 13.7
+    .. versionchanged:: 20.0
+       Argument and attribute ``can_manage_voice_chats`` where renamed to
```

Comment:
```suggestion
       Argument and attribute ``can_manage_voice_chats`` were renamed to
```

double check for other places